site/content/blog/2019-12-20-onboarding-with-mattermost.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Onboarding with Mattermost"
+slug: onboarding-with-mattermost
+date: 2019-12-20T00:00:00-04:00
+categories:
+    - "announcements"
+author: CJ Wild
+github: imisshtml
+community: claude.wild
+---
+
+I recently joined Mattermost as a Remote Software Engineer and while I’ve worked in remote environments before, I was really surprised at how amazing this company is to work with. During my onboarding I discovered they have really developed a culture built around being Remote; one that allowed for me to feel as part of a larger team and always engaged, rather than separated from my peers and working alone. 
```

Comment:
```suggestion
I recently joined Mattermost as a Remote Software Engineer and while I’ve worked in remote environments before, I was really surprised at how amazing this company is to work with. During my onboarding I discovered they have really developed a culture built around being remote; one that allowed for me to feel as part of a larger team and always engaged, rather than separated from my peers and working alone. 
```